moveit_planners/stomp/stomp_moveit/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package>
+  <name>stomp_moveit</name>
+  <version>0.0.0</version>
+  <description>The stomp_moveit package</description>
+
+  <maintainer email="jrgnichodevel@gmail.com">Jorge Nicho</maintainer>
+  <license>Apache 2.0</license>
```

Comment:
Many parts of this package (the parts from Willow) are still BSD-3, so this needs to be listed as well. It would be really nice if SwRI could agree to change all of the licenses to BSD-3 so that moveit remains consistent in its codebase